lib/models/user.model.js

```diff
@@ -1,6 +1,10 @@
 const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
-
+const contact = new Schema({
+	fullName: { type:String, required:true },
+	avatarURL:{ type:String, required:true },
```

Comment:
put a space

Reply:
where exactly?

Reply:
Can't you see???? It is avatarURL:{} it should be avatarURL: {} hahaha :D Just testing comments back and forth. 👍